site/vite.config.ts

```diff
@@ -162,7 +161,7 @@ function monEntrepriseDevServer(): Plugin {
 		configureServer() {
 			// We could use native ViteJS watch API, but it would require changing
 			// more code and maybe the whole "modele-social" package build process.
-			watchDottedNames()
+			// watchDottedNames()
```

Comment:
Une possibilité : https://github.com/jycouet/kitql/tree/main/packages/vite-plugin-watch-and-run

Reply:
Voire on peut réimplémenter la logique sans passer par ce plugin

- Ajouter les fichiers à suivre https://vitejs.dev/config/#server-watch
- Évenemement `watcher.on` https://github.com/jycouet/kitql/blob/4c291355486fd4e016d2c64f7b52c7bfa8c27f5d/packages/vite-plugin-watch-and-run/src/index.ts#L119-L121